app/main.py

```diff
@@ -1,3 +1,6 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    for char in phrase:
+        count = 0
```

Comment:
The `count` variable is initialized to 0 inside the loop. This causes the count to reset to 0 for each character in the phrase, resulting in the function always returning 0 or 1. To fix this, initialize `count` to 0 before the loop starts.